pdh_check/check.py

```diff
@@ -0,0 +1,67 @@
+# (C) Datadog, Inc. 2013-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# project
+from checks import AgentCheck
+from utils.containers import hash_mutable
+# datadog
+try:
+    from checks.libs.win.winpdh import WinPDHCounter
+except ImportError:
+    def WinPDHCounter(*args, **kwargs):
```

Comment:
Do we really want to do this? Don't we prefer to raise? It shouldn't really happen right?